data/dev-server.handlebars

```diff
@@ -0,0 +1,14 @@
+<html>
+
+<head>
+  {{{head}}}
+  {{{css}}}
+</head>
+
+<body>
+  <div id="react-root">
+    {{{body}}}
+  </div>
+</body>
+
+</html>
```

Comment:
Not sure which templating language is the simplest, just went with handlebars for now, but absolutely not married to that.

Reply:
I actually wondered if it were possible to use React components for this, but currently Stitch doesn’t accept that.

Reply:
Being that stitch already uses handlebars for stuff, I think this is good. 👍